aplib.net-demo/Assets/Scripts/Items/Items/TestItem.cs

```diff
@@ -0,0 +1,18 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class TestItem : Item
+{
+
+    //You want to change the value of the uses and all the attributes of the item in the unity inspector for the item prefab you make
+
+    /// <summary>
+    /// Currently does nothing, this may be where we want to add effect that last a bit longer than just a single instant
+    /// </summary>
+    void Update()
+    {
+        
+    }
```

Comment:
Why does this function exist if it does nothing? Why does the comment suggest some implementation?

Reply:
Testitems will be removed in final commit